assignment/tool/src/webapp/vm/assignment/assignment_macros.vm

```diff
@@ -411,24 +411,25 @@
         #set($dynWidth = 100.0 / $!progressStatus.size())
         #set($totalWidthBg = 0)
         #set($statusSet = $!progressStatus.keySet())
-        <div class="progress-container">
+        #set($currentStatus = $tlang.getString("gen.notsta"))
+        <div tabindex="0" class="progress-container" role="status" aria-labelledby="status">
             <div class="progress progress-cols">
                 #foreach ($status in $!statusSet.iterator())
+                    <div class="progress-bar progress-bar-empty" style="width: $dynWidth%">$status</div>
                     #if($!progressStatus.get($status))
-                        <div class="progress-bar progress-bar-empty progress-bar-active" role="progressbar" style="width: $dynWidth%" aria-valuenow="100" aria-valuemin="0" aria-valuemax="$dynWidth">$status</div>
                         #set($totalWidthBg = $totalWidthBg + $dynWidth)
-                    #else
-                        <div class="progress-bar progress-bar-empty" role="progressbar" style="width: $dynWidth%" aria-valuenow="100" aria-valuemin="0" aria-valuemax="$dynWidth">$status</div>
+                        #set($currentStatus = $status) ## Saves the last status of the hashmap that equals true, used for screen reader
                     #end
                 #end
             </div>
+            <div id="status" hidden>$currentStatus</div>
             <div class="progress">
                 #set($minWidth = $totalWidthBg - $dynWidth)
-                <div class="progress-bar progress-bar-striped active" role="progressbar" style="width: $minWidth%" aria-valuenow="100" aria-valuemin="0" aria-valuemax="$totalWidthBg"></div>
```

Comment:
Removed `role` and `aria` attr. as weren't being used (the progress bar wasn't focusable). Anyway I think it isn't useful for this case.